netlify/netlify-ipx
# feat: support custom response headers

* Update README instructions on developing locally
* Update example project with what custom response headers could look like. The data structure used is the same format expected from [custom headers](https://docs.netlify.com/routing/headers/) specified within a `netlify.toml` in a project

Testing instructions:
* Run the example app
* In some of the image requests, you should be able to see the HSTS and `X-Test` headers being set in the response
